lib/rules/declaration-block-no-redundant-longhand-properties/README.md

```diff
@@ -60,6 +60,10 @@ This rule complains when the following shorthand properties can be used:
 -   `text-emphasis`
 -   `mask`
 
+**Please note** that properties are considered to be redundant if they may be written shorthand according to the specification, **regardless of the behavior of any individual browser**. For example, due to Internet Explorer's implementation of Flexbox, [it may not be possible to use the shorthand property `flex`](https://github.com/philipwalton/flexbugs#flexbug-8), but the longhand form is still considered a violation. 
+
+Flexbox-related properties can be ignored using `ignoreShorthands: ["/flex-/"]` (see below).
```

Comment:
It can be just as `"flexbox"`, without `Regexp`, doesn't it?

Reply:
The documentation is a bit confusing as "padding" is given as an example, but the option is `ignoreShorthands`, and the shorthand properties are `flex-basis`, `flex-grow` and `flex-shrink`. I think I'd have to look at the code to figure out what the actual behaviour is. In any case despite being a regex, this seems more straightforward.